pyOCD/pyDAPAccess/dap_access_cmsis_dap.py

```diff
@@ -360,14 +365,35 @@ def get_connected_devices():
     @staticmethod
     def get_device(device_id):
         assert isinstance(device_id, str)
-        return DAPAccessUSB(device_id)
+        return DAPAccessCMSISDAP(device_id)
+
+    @staticmethod
+    def set_args(arg_list):
+        # arg_list =[['ws_host=localhost', 'ws_port=8081']]
+        # Match ws_host=localhost
+        arg_pattern = re.compile("([^=]+)=(.*)")
+        for arg in arg_list[0]:
```

Comment:
When pyocd-gdbserver is run without setting any dapargs (-da DAPARG) I get an exception.